ktx-library-codelab/step-06/myktxlibrary/src/main/java/com/example/android/myktxlibrary/LocationUtils.kt

```diff
@@ -58,7 +58,7 @@ fun FusedLocationProviderClient.locationFlow() = callbackFlow<Location> {
             result ?: return
             for (location in result.locations) {
                 try {
```

Comment:
I think with `trySend` instead of `offer` the try/catch is not needed, as trySend will not throw an exception

Reply:
I meant removing the try/catch and just leaving what's inside, like this:
```
override fun onLocationResult(result: LocationResult?) {
  result ?: return
  for (location in result.locations) {
     trySend(location)
  }
}
```

Reply:
Done.